CONTRIBUTING.md

```diff
@@ -0,0 +1,2 @@
+# Contributing
+Submit a pull request or something.
```

Comment:
Should we have a contributor agreement?

Reply:
IMO not necessary, the GPL is implicit agreement for all the scenarios I can think of (which aren't many).

@ktrieu thoughts?

Reply:
Yeah, I think we'll be OK without one for now.

Reply:
Yeah, I think we'll be OK without one for now.